translations/data/home.json

```diff
@@ -1,125 +1,143 @@
         "ar": "هذا هو بالضبط ما يتيح لك الفديفيرس القيام به."
     },
     "intro_btn_interesting": {
         "en-us": "Interesting",
+	"de-de": "Interessant.",
         "sk": "Zaujímavé",
         "fr": "Intéressant !",
         "ru": "Интересно",
         "ar": "ممتع"
     },
     "intro_fediverse_is": {
         "en-us": "Fediverse is a common name for social media sites that <mark>cooperate</mark> instead of competing.",
+	"de-de": "Fediverse ist ein Oberbegriff für soziale Medien, die zusammenarbeiten statt zu konkurrieren.",
         "sk": "Fediverse je sieť sociálnych médií, ktoré <mark>spolupracujú</mark> namiesto toho, aby súťažili.",
         "fr": "Le fediverse est un nom générique pour les réseaux sociaux qui <mark>coopèrent</mark>, plutôt que de se faire concurrence.",
         "ru": "Fediverse это общее название для социальных сетей, которые <mark>сотрудничают</mark> вместо того, чтобы конкурировать.",
         "ar": "الفديفيرس هو اسم شائع لمواقع التواصل الاجتماعي التي <mark> تتعاون </ mark> بدلاً من المنافسة."
     },
     "intro_fediverse_social_media": {
         "en-us": "And every major social media site has a great <mark>alternative</mark> in the fediverse, so it doesn't matter if you're into sharing images, videos, news, or just short quips. You will find <mark>your home</mark>.",
-        "sk": "A väčšina má skvelú <mark>alternatívu</mark> vo fediverse, takže nezáleží na tom, či chceš zdieľať obrázky, videá, správy, alebo len krátke vtipy. Nájdeš tu <mark>svoj domov</mark>.",
+	"de-de": "Und zu jeder wichtigen Social Media Seite gibt es im Fediverse tolle <mark>Alternativen</mark>. Es kommt also nicht darauf an, ob Du Bilder, Videos, Nachrichten oder witzige Sprüche austauschen möchtest. Du wirst ein passendes Zuhause finden.",
+	 "sk": "A väčšina má skvelú <mark>alternatívu</mark> vo fediverse, takže nezáleží na tom, či chceš zdieľať obrázky, videá, správy, alebo len krátke vtipy. Nájdeš tu <mark>svoj domov</mark>.",
         "fr": "Et chaque réseau social majeur dispose d'une super <mark>alternative</mark> dans le fediverse, donc peu importe que vous préfériez partager des images, des vidéos, des informations ou de courts messages. Vous trouverez ce qui <mark>vous convient le mieux</mark>.",
         "ru": "И каждая популярная социальная сеть имеет отличную <mark>альтернативу</mark> в fediverse, так что неважно, нравится ли вам делиться картинками, видео, новостями, или просто колкими шутками.",
         "ar": "وكل موقع من مواقع التواصل الاجتماعي الرئيسية لديه <mark> بديل </ mark> رائع في الفديفيرس، لذلك لا يهم إذا كنت تشارك الصور أو مقاطع الفيديو أو الأخبار أو مجرد المزح القصيرة. سوف تجد <mark> منزلك </ mark>."
     },
     "intro_btn_alright": {
         "en-us": "Alright!",
+	"de-de": "Zauberhaft !",
         "sk": "Super!",
         "fr": "Génial !",
         "ru": "Отлично!",
         "ar": "ممتاز!"
     },
     "outro_join": {
         "en-us": "Join the fediverse",
+	"de-de": "Komm ins Fediverse !",

```

Comment:
Same here, maybe on more places, too?

Reply:
The ! should be removed as this is a split sentence, to avoid having to add HTML of the link in here.

![[Join the fediverse](https://jointhefediverse.net/join) or [see how it works](https://jointhefediverse.net/learn)](https://github.com/jointhefediverse-net/jointhefediverse.net/assets/3909390/2a5c5490-9aa8-4c3a-b274-c768017b7c40)